stock-app/src/main/java/edu/JasonFlinkstrom/advancedjava/stock/parseDate.java

```diff
@@ -0,0 +1,26 @@
+package edu.JasonFlinkstrom.advancedjava.stock;
+
+import javax.validation.constraints.NotNull;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Converts a String to a Calendar
+ */
+public class parseDate {
+

```

Comment:
classes always start with upper case
class names are typically nouns not verbs.